Gruntfile.js

```diff
@@ -143,7 +143,7 @@ Z.fun = {
         "./bower_components/angular-uuid2/dist/angular-uuid2.min.js",
         //"./vendor/angular-foundation-colorpicker/js/foundation-colorpicker-module.min.js",
         "./vendor/uservoice/uservoice.min.js",
-        //"./vendor/angularJS-toaster/toaster.min.js",
+        "./vendor/angularJS-toaster/toaster.min.js",
```

Comment:
After rebasing off of `gardener_widget` I had to add this back in. From what I can tell, it didn't cause any issues previously because `dist/arethusa_packages.min.js` was not being copied, but now that `dist/arethusa_packages.min.js` is being copied into the client application again, it fails with this line removed.

Reply:
Ok sorry. Will need to figure out how to fix the source map error another time